pkg/log/locallogger/locallogger.go

```diff
@@ -18,7 +19,7 @@ type localLogger struct {
 	logger log.Logger
 }
 
-func NewKitLogger(logFilePath string) log.Logger {
+func NewKitLogger(logFilePath string) (log.Logger, io.Writer) {
```

Comment:
This feels like a weird change.

Reply:
yea, i droped it and just expose via Writer() func